frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -100,11 +108,36 @@ export class ClusterOverview extends React.Component {
         },
         loaded: false,
       },
+      capacityStats: {
+        stats: {
+          cpu: {
+            title: 'CPU',
+            unit: 'cores',
+            data: {},
+          },
+          memory: {
+            title: 'Memory',
```

Comment:
This is how things should be rendered. I think it makes sense to pass the raw data to the dashboard and process this there.